### PR TITLE
Adding tutorial page for Pong videos

### PR DESCRIPTION
**Pull request by erickson-doug**
Signed-off-by: Doug Erickson <dougeric@amazon.com>

### Submission Checklist:

* [ ] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [ ] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [ ] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [ ] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/docs/learning-guide/_index.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ This guide contains video tutorials, written step-by-step tutorials, and sample
 
 | Section | Description |
 | - | - |
+| [Create your first O3DE game: Pong](first-project/) | Start from a new O3DE project and build your first game: Pong! What's "Pong"? Two paddles square off, in a ball-based battle to land the highest score. This tutorial series will introduce you to basic game UI development, scripting, and controls in O3DE. |
```
Comment (willihay) [inline content/docs/learning-guide/_index.md:16]:
```suggestion
| [Create your first O3DE game: Pong](tutorials/first-project/) | Start from a new O3DE project and build your first game: Pong! What's "Pong"? Two paddles square off, in a ball-based battle to land the highest score. This tutorial series will introduce you to basic game UI development, scripting, and controls in O3DE. |
```

File: content/docs/learning-guide/tutorials/first-project/_index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+linkTitle: PostFX
```
Comment (willihay) [inline content/docs/learning-guide/tutorials/first-project/_index.md:2]: copy-paste lolz
```suggestion
linkTitle: Pong Game
```

File: content/docs/learning-guide/tutorials/first-project/_index.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+title: Create a Simple Pong Game with O3DE
+description: Videos to guide you through creating a basic Pong game in Open 3D Engine (O3DE).
+menu_uuid: Your First Project
```
Comment (willihay) [inline content/docs/learning-guide/tutorials/first-project/_index.md:5]: This metadata is not needed:
```suggestion
```

File: content/docs/learning-guide/tutorials/first-project/_index.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+toc: true
+---
+
+This topic contains video tutorials that will guide you through creating a Pong game in Open 3D Engine (O3DE), from creating a new project to defining gameplay state and user interface elements.
+
+## Video Tutorial Series: Create the Pong Game with Open 3D Engine (O3DE)
```
Comment (willihay) [inline content/docs/learning-guide/tutorials/first-project/_index.md:11]: You've already introduced the abbreviation in the intro paragraph.
```suggestion
## Video Tutorial Series: Create the Pong Game with O3DE
```

File: content/docs/learning-guide/tutorials/first-project/_index.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+---
+linkTitle: PostFX
+title: Create a Simple Pong Game with O3DE
+description: Videos to guide you through creating a basic Pong game in Open 3D Engine (O3DE).
+menu_uuid: Your First Project
+toc: true
+---
+
+This topic contains video tutorials that will guide you through creating a Pong game in Open 3D Engine (O3DE), from creating a new project to defining gameplay state and user interface elements.
```
Comment (willihay) [inline content/docs/learning-guide/tutorials/first-project/_index.md:9]: Latest style we've been using:
```suggestion
This topic contains video tutorials that will guide you through creating a Pong game in **Open 3D Engine (O3DE)**, from creating a new project to defining gameplay state and user interface elements.
```

File: content/docs/learning-guide/tutorials/first-project/_index.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+## Video Tutorial Series: Create the Pong Game with Open 3D Engine (O3DE)
+
+Games, as conventional wisdom goes, started with Pong. Your experience with O3DE can start with Pong, too! Let AWS GameTech's technical trainer, Alex Damarjian, guide you through learning the basics of O3DE game development with this series, while letting you relive history with a modern game engine and tools.
```
Comment (willihay) [inline content/docs/learning-guide/tutorials/first-project/_index.md:13]: Unlike GameLift, they spell it "Game Tech", with a space.
```suggestion
Games, as conventional wisdom goes, started with Pong. Your experience with O3DE can start with Pong, too! Let AWS Game Tech's technical trainer, Alex Damarjian, guide you through learning the basics of O3DE game development with this series, while letting you relive history with a modern game engine and tools.  
```

File: content/docs/learning-guide/tutorials/first-project/_index.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+toc: true
+---
+
+This topic contains video tutorials that will guide you through creating a Pong game in Open 3D Engine (O3DE), from creating a new project to defining gameplay state and user interface elements.
+
+## Video Tutorial Series: Create the Pong Game with Open 3D Engine (O3DE)
+
+Games, as conventional wisdom goes, started with Pong. Your experience with O3DE can start with Pong, too! Let AWS GameTech's technical trainer, Alex Damarjian, guide you through learning the basics of O3DE game development with this series, while letting you relive history with a modern game engine and tools.  
+
+[Take me to the full playlist!](https://www.youtube.com/watch?v=kK0XnYFKw38&list=PLCQwFpnHSZQhhsr6iJqyeYqKe6dexnHh9)
```
Comment (willihay) [inline content/docs/learning-guide/tutorials/first-project/_index.md:15]: Technically, your link takes you to the first video in the series, not the playlist. Here's the playlist link:
```suggestion
[Take me to the full playlist!](https://www.youtube.com/playlist?list=PLCQwFpnHSZQhhsr6iJqyeYqKe6dexnHh9)
```

File: content/docs/welcome-guide/_index.md [hunk 8]
```diff
@@ -25,6 +25,7 @@ O3DE is an open-source, cross-platform, real time 3D engine that you can use to
 | [Set up O3DE](setup/) | Install O3DE and set up your environment. |
 | [Project creation](create/) | Create your first O3DE project. |
 | [Workflow tours](tours/) | Get a closer look at the common areas of interest in O3DE for each creative role on a project. Take a quick tour of O3DE Editor-the primary workspace in O3DE. |
+| [Create your first game: Pong!](/tutorials/first-project/)
```
Comment (willihay) [inline content/docs/welcome-guide/_index.md:28]:
```suggestion
| [Create your first game: Pong](/docs/learning-guide/tutorials/first-project/) | Start from a new O3DE project and build your first game: Pong! What's "Pong"? Two paddles square off, in a ball-based battle to land the highest score. This tutorial series will introduce you to basic game UI development, scripting, and controls in O3DE. |
```

**Review by willihay — CHANGES_REQUESTED** [review]

**Review by willihay — APPROVED** [review]

**Review by adamarjian1 — APPROVED** [review]

**Pull request merged**